_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -58664,6 +58672,14 @@
 /obj/effect/decal/cleanable/dirt,
 /turf/open/floor/plating,
 /area/maintenance/fore/secondary)
+"uym" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 4
+	},
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/spawner/structure/window/reinforced,
+/turf/open/floor/iron,
+/area/maintenance/disposal)
```

Comment:
where is this? it shows a normal r-window, not a directional one, when only directional windows exist in disposals.

Reply:
![image](https://user-images.githubusercontent.com/85910816/133661184-ca449bea-b699-4baa-9a7e-a929fbf36278.png)
I just thought this looked nice and not unnatural because the button is needed for the door

Reply:
![image](https://user-images.githubusercontent.com/85910816/133661410-c1beab8a-effd-4616-bb6e-4368070f0c14.png)

would this fit the style better?